1993 - Fixed an issue where cap modal was open only one time [4.18.x] (#1998)
infor-design/enterprise: docs/CHANGELOG.md

- `[Busy Indicator]` Fixed a bug that causes a javascript error when the busy indicator is used on the body tag. ([#1918](https://github.com/infor-design/enterprise/issues/1918))
- `[Css/Sass]` Fixed an issue where the High Contrast theme and Uplift theme were not using the right tokens. ([#1897](https://github.com/infor-design/enterprise/pull/1897))
- `[Colors]` Fixed the color palette demo page to showcase the correct hex values based on the current theme ([#1801](https://github.com/infor-design/enterprise/issues/1801))

> - `[Contextual Action Panel]` Fixed an issue where cap modal was open only one time. ([#1993](https://github.com/infor-design/enterprise/issues/1993))

Reviewer: ```suggestion
- `[Contextual Action Panel]` Fixed an issue where cap modal would only open the first time. ([#1993](https://github.com/infor-design/enterprise/issues/1993))
```
